core/vm/errors.go

```diff
@@ -36,6 +36,7 @@ var (
 	ErrGasUintOverflow          = errors.New("gas uint64 overflow")
 	ErrInvalidCode              = errors.New("invalid code: must not begin with 0xef")
 	ErrNonceUintOverflow        = errors.New("nonce uint64 overflow")
+	ErrInvalidOpcode            = errors.New("exection reached INVALID (0xfe) opcode")
```

Comment:
```suggestion
	ErrInvalidOpcode            = errors.New("execution reached INVALID (0xfe) opcode")
```

Reply:
Could also just say:
```suggestion
	ErrInvalidOpcode            = errors.New("aborted with INVALID (0xfe) opcode")
```

Reply:
Lastly with [EIP-1803](https://eips.ethereum.org/EIPS/eip-1803) I hope we can rename `INVALID` to `ABORT` to make it more clear, but that should be a separate PR I suppose?